projects/ion/src/lib/input-area/input-area.component.html

```diff
@@ -0,0 +1,6 @@
+<textarea
+  data-testid="input-area"
+  [cols]="cols"
+  [rows]="rows"
+  [attr.placeholder]="placeholder"
+></textarea>
```

Comment:
As I told you on chat, you'll need to treat value and value changes in this component too, similar of what we have on input component.